app/controllers/application_controller.rb

```diff
@@ -1,6 +1,8 @@
 class ApplicationController < ActionController::Base
+  include Pundit
```

Comment:
Great job on the clean application controller! Great readability as well as functionality.